lib/constants.js

```diff
@@ -12,3 +13,9 @@ export const BLOCKED_PAGES = [
   '/_app',
   '/_error'
 ]
+export const IS_BUNDLED_PAGE = /^bundles[/\\]pages.*\.js$/
+export const MATCH_ROUTE_NAME = /^bundles[/\\]pages[/\\](.*)\.js$/
```

Comment:
It's not obvious that these are Regexs. what do you think about `BUNDLE_PAGE_REGEX` and `ROUTE_NAME_REGEX`?